lib/runner/util.js

```diff
@@ -61,5 +61,44 @@ module.exports = {
             }
         }
         return dest;
+    },
+
+    createReadStream: function (resolver, fileSrc, callback) {
+        // bail out in case of invalid fileSrc or resolver.
+        if (!fileSrc || typeof fileSrc !== 'string') {
+            return callback(new Error('missing file source'));
+        }
+        else if (!resolver || typeof resolver.stat !== 'function' || typeof resolver.createReadStream !== 'function') {
+            return callback(new Error('invalid file resolver'));
+        }
+
+        resolver.stat(fileSrc, function(err, stats) {
+            if (err) {
+                // overwrite `ENOENT: no such file or directory` error message. Most likely the case.
+                err.code === 'ENOENT' && (err.message = `"${fileSrc}", no such file`);
+                return callback(err);
+            }
+
+            var stream;
+
+            // check for a valid file.
+            if (!stats.isFile()) {
```

Comment:
since resolver is externally provided, don't assume any of it's behaviour. check truth `stats` and isFunction `isFile`